### ISSUE TITLE
Move "Save" button in the new reminder dialog to the right of the "Remind me about" input field

### ISSUE THREAD

**Issue by rryk** [vermayash8/TaskNearby#82]
At the current location (bottom of the activity), it is hidden by the keyboard that appears when entering reminder description and thus to save the reminder one has to first close the keyboard and then click on the "Save" button. Moving it to the right of the input text as in many other apps, would allow to save the reminder without having to close the keyboard.

**Comment by rryk** [comment]
An alternative would be to convert input field to a single-line text and change the "new line" key in the keyboard to one that saves the reminder. Long multiple descriptions would still be possible using the Notes field.

**Comment by vermayash8** [comment]
> At the current location (bottom of the activity), it is hidden by the keyboard that appears when entering reminder description and thus to save the reminder one has to first close the keyboard and then click on the "Save" button. Moving it to the right of the description field on the top, would allow to save the reminder without having to close the keyboard.

I agree with the suggestion with respect to the issue you're facing. But, we'll have to check if we can do that without compromising on UX. Context: Most of the fields are present at the bottom (e.g. Location input, alarm on/off, attachment, schedule etc.). Moving this to the top might cause customers to miss those as well.

But yeah, your point makes sense that we need to try and experiment with a different position of the button. We'll try to take this up in our triaging plan.

> An alternative would be to convert description field on the top to a single-line text and change the "new line" key in the keyboard to one that saves the reminder. Long multi-line descriptions would still be possible using the Notes field.

I'll not prefer doing this considering the flow in which a user adds a task. To create a task, two things are really essential - task name and location. If we put in the task name's enter key to be save, that breaks the CX because customer will see the error - "Please select a location.".

**Comment by rryk** [comment]
> I'll not prefer doing this considering the flow in which a user adds a task. To create a task, two things are really essential - task name and location. If we put in the task name's enter key to be save, that breaks the CX because customer will see the error - "Please select a location.".

That's a good point. Thank you for considering though.
